src/widget/widget.cpp

```diff
@@ -931,34 +931,60 @@ void Widget::reloadHistory()
     }
 }
 
+/**
+ * @brief Plays a sound via the audioNotification AudioSink
+ * @param sound Sound to play
+ * @param loop if true, loop the sound until onStopNotification() is called
+ */
+void Widget::playNotificationSound(IAudioSink::Sound sound, bool loop) {
```

Comment:
What do you think about using `enum class` instead of `bool`?
Because not quite clear, that is the difference between
```
playNotificationSound(sound, true);
playNotificationSound(sound, false);
```

But much better
```
playNotificationSound(sound, LoopSound);
playNotificationSound(sound, DoNotLoopSound);
```